mysqlcluster/mysqlcluster.go

```diff
@@ -114,12 +118,19 @@ func (c *MysqlCluster) GetSelectorLabels() labels.Set {
 
 // GetMySQLVersion returns the MySQL server version.
 func (c *MysqlCluster) GetMySQLVersion() string {
-	version := c.Spec.MysqlVersion
-	// lookup for an alias, usually this will solve 5.7 to 5.7.x
-	if v, ok := utils.MySQLTagsToSemVer[version]; ok {
+	var version string
+	// lookup for an alias, this will solve MySQL tags:
+	// 5.7 --> 5.7.x
+	// 8.0 --> 8.0.x
```

Comment:
Delete the line. When support 8.0, add the comment.

Reply:
Done.